test-kuttl/e2e/syncthing-cluster-sync/README.md

```diff
@@ -0,0 +1,30 @@
+# Syncthing Sync
```

Comment:
The e2e has quite the set of inline scripts... For items that are common across scripts, you could have a common script that gets `source`-d. That might make the individual test steps a bit more readable.
Note: I haven't tried this, so I'm not positive this will work